src/themes/money/theme.json

```diff
@@ -1802,6 +1802,27 @@
           "display": "flex",
           "alignItems": "center"
         },
+        "assurance-text": {
+          "boxSizing": "border-box",
+          "px": ["lg", "56px"],
+          "py": ["lg", "40px"],
+          "mt": ["md", "lg"],
+          "mb": "md",
+          "mx": "auto",
+          "textAlign": "left",
+          "backgroundColor": "grey-05",
+          "maxWidth": "720px",
+          "&& div": {
```

Comment:
What is `&&`?

Reply:
It's there to increase the specificity (puts the generated class twice, doubling the rule specificity) to allow overriding the background color from the parent palette, otherwise the background of all child `div`s are not transparent if there's a palette above.

Reply:
👍  Interesting, I did not know you could do that with CSS